src/index.ts

```diff
@@ -1,7 +1,9 @@
+import 'reflect-metadata'; // shim required for routing-controllers
+import 'module-alias/register';
 import express from 'express';
 import cors from 'cors';
```

Comment:
It doesn't look like we're using cors anymore in this file so we can probably remove this import statement.

Reply:
yep - will do. have a lot of stuff I haven't had the time to cleanup yet